frontend/src/concepts/pipelines/content/tables/pipelineRun/PipelineRunTable.tsx

```diff
@@ -32,6 +32,7 @@ const PipelineRunTable: React.FC<PipelineRunTableProps> = ({ runs }) => {
         data={filteredRuns}
         columns={pipelineRunColumns}
         enablePagination
+        minPageSize={5}
```

Comment:
Don't do this -- data should be limited if we need to truncate the data... don't force the tables to be smaller.